framework/LoopMeUnitedSDK/Core/Internal/LoopMeSDK.m

```diff
@@ -100,22 +100,8 @@ - (void)init: (LoopMeSDKConfiguration *)configuration completionBlock: (void(^_N
     [[LoopMeGDPRTools sharedInstance] getAppDetailsFromServer];
     [[LoopMeGDPRTools sharedInstance] prepareConsent];
     [LoopMeGlobalSettings sharedInstance];
-    // TODO: Why we do not store OMID Wrapper instance?
-    (void)[LoopMeOMIDWrapper initOMIDWithCompletionBlock: ^(BOOL ready) {
-        if (self.isReady && ready) {
-            return;
-        }
-        self.isReady = ready;
-        NSError *error;
-        NSString *description = @"LoopMe OMID fail to initialize";
-        if (!ready) {
-            [LoopMeErrorEventSender sendError: LoopMeEventErrorTypeServer errorMessage: description appkey: @"unknown"];
-            error = [NSError errorWithDomain: @"loopme.com" code: -1 userInfo: @{NSLocalizedDescriptionKey: description}];
-        }
-        if (completionBlock != nil) {
-            completionBlock(ready, error);
-        }
-    }];
+    self.isReady = YES;
+    completionBlock(YES, nil);
```

Comment:
Here we have value `false ` in  completionBlock 
` if (completionBlock != nil) {
            completionBlock(false, [NSError errorWithDomain: @"loopme.com" code: 0 userInfo: userInfo]);
        }`
        
       and here `YES` 
        
        `completionBlock(YES, nil);`
        
        Maybe better keep similar? YES/NO or  true/false

Reply:
The block you reference fails initialization. When calling the completion block with YES, the SDK is initialized.

The use of `true/false` is discouraged in Obj-C. `YES/NO` is the preferred syntax.

Reply:
See here what our adapter expects:
https://github.com/loopme/ios-united-sdk/blob/master/Mediation/IronSource/ISLoopMeCustomAdapter/ISLoopmeCustomAdapter.m#L28